### DISCUSSION TITLE
DHCP Server issue. Raspberry Pi OS Bookworm static IP not detected

### DISCUSSION THREAD

**Discussion by mjh17474** [AdguardTeam/AdGuardHome#6467] [General]
I thought I'd give AdGuardHome ago with the view of replacing Pi-Hole which is also my DHCP server.
After configuring, I realised I couldn't enable the DHCP server as AdGuard hadn't detected the IP as being static.
I'm guessing as networking has changed again in the new Pi OS (Bookworm), it has broken how AdGuard detects a static IP.

I started from fresh on Pi OS Bullseye and it all worked perfectly and detected the IP as static.

I've googled around and found similar issues relating to docker instances and FreeBSD but nothing that has helped fix the problem on Pi OS Bookworm. 

I'm guessing this is something that will be catered for in a future release?

**Comment by Eirikr70** [comment]
Same problem here.
